README.md

```diff
@@ -77,13 +77,71 @@ Bootstrapping on Ubuntu requires sudo privileges to install all the required dep
 
 NOTE: A reboot is required after running the bootstrap command on Ubuntu.
 
-For all other processes (build, start etc.) the commands are same for Ubuntu and Mac OS.
+For all other processes (build, start etc.) the commands are the same for Ubuntu and Mac OS.
 
 Installed tools include: [git 2](https://git-scm.com),
 [maven 3](https://maven.apache.org), [OpenJdk 8](http://openjdk.java.net/projects/jdk8/),
 [docker 1.12](https://docker.com), [node 6](https://nodejs.org/en) and [python 2.7](https://www.python.org)
 
-Similar tooling will work for Windows. Contribute a patch :)
+
+##### Windows (7+)
+
+To install Wasabi's dependencies on Windows we use [Chocolatey][win_choco] which
+needs administrator rights in your [cmd.exe][win_cmd_admin].
+
+If you have git, just run:
+```dos
+git clone https://github.com/intuit/wasabi.git
```

Comment:
maybe streamline with unix/osx format of $ in front? I just used this without much thinking as I like easily copyable text...